docs/schema/datasource-element-datasourcestype-complextypeaction4.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: DataSource element
+description: Learn about the DataSource element.
 manager: soliver
-ms.date: 3/9/2015
+ms.date: 6/9/2022
```

Comment:
This needs to be in the MM/DD/YYYY format.

Reply:
Addressed.